documents/alembic_migrations.md

```diff
@@ -0,0 +1,150 @@
+# Database Migration System with Alembic and Docker
+
+This document provides instructions for setting up and managing database migrations in the `biotablero-search-backend` project using Alembic in a Docker environment. It includes the purpose of `init_postgis.sql` and essential Alembic commands.
+
+## Setup and Configuration
+
+### Environment Configuration
+
+Before running the Docker setup, create a `.env` file in the project root to configure your database credentials. Alembic and other components require these environment variables for proper functionality.
+
+Here is an example of the `.env` file:
+
+```text
+POSTGRES_USER=your_postgres_user
+POSTGRES_PASSWORD=your_postgres_password
+POSTGRES_DB=your_database_name
+
+DATABASE_URL_SYNC=postgresql://your_postgres_user:your_postgres_password@localhost/your_database_name
+DATABASE_URL_ASYNC=postgresql+asyncpg://your_postgres_user:your_postgres_password@localhost/your_database_name
+```
```

Comment:
I'm not sure if we should avoid using the environment variables DATABASE_URL_SYNC and DATABASE_URL_ASYNC. Instead, we could build the connection strings dynamically using the existing environment variables (POSTGRES_USER, POSTGRES_PASSWORD, POSTGRES_DB) and store it in a global variable or something similar.

Let me know your thoughts on this, @erikasv @ManuelStardust.

Reply:
I agree, and you should create an environment variable for the database host

Reply:
You didn't include this change and it is important

Reply:
In addition, please declare a new env var with the db port and change the docker-compose file to avoid hardcode the port.